src/pages/resultsView/survival/SurvivalUtil.tsx

```diff
@@ -25,6 +26,13 @@ export type GroupedScatterData = {
     [key: string]: SurvivalCurveData;
 };
 
+export type SurvivalSummary = {
+    estimate: number;
```

Comment:
estimate of what?   maybe a more descriptive name.  like somethingEstimate?

Reply:
This is a estimate of [Survival Function](https://en.wikipedia.org/wiki/Survival_function), maybe change to `survivalFunctionEstimate` makes more sense?